java/bogdana_maksiuta/src/com/zoolatech/lecture4/tasks/_7/NumberInWord.java

```diff
@@ -0,0 +1,39 @@
+package com.zoolatech.lecture4.tasks._7;
+
+/**
+ * 7. Create a method that accepts a string of digits and
+ * after each digit appends its string representation in parentheses
+ * Input: “12345”
+ * Output: “1(one)2(two)3(three)4(four)5(five)”
+ */
+public class NumberInWord {
+    void parsing(int value) {
+        StringBuilder newLine = new StringBuilder();
+        for (int i = 0; i < String.valueOf(value).length(); i++) {
+            char number = String.valueOf(value).charAt(i);
+
+            switch (number) {
```

Comment:
you could remove the code duplication (`newLine.append`) with the help of the switch expression